src/content/en/fundamentals/security/credential-management/index.md

```diff
@@ -32,21 +32,16 @@ and take a look at the
 ### Check Credential Management API browser support
 
 Before using the Credential Management API,
-first check the API is supported,
-and that the version of the API is compatible with your code.
+first check the API is supported.
 
-[Updates to the Credential Management API](/web/updates/2017/06/credential-management-updates)
-landed in Chrome 60.
-These updates contain backward incompatible changes.
-To check that your implementation isn't triggered in older versions of the API,
-see if `preventSilentAccess exists`:
-
-    if (navigator.credentials && navigator.credentials.preventSilentAccess) {
-      // The new Credential Management API is available
+    if (window.PasswordCredential || window.FederatedCredential) {
+      // The Credential Management API is available
     }
 
-For more information, see the
-[Chrome 60 migration guide](https://docs.google.com/document/d/154cO-0d5paDFfhN79GNdet1VeMUmELKhNv3YHvVSOh8).
+Warning: Feature detection by checking `navigator.credentials` may break your
+website on browsers supporting 
+[WebAuthn(PublicKeyCredential)](https://www.w3.org/TR/webauthn/) but the
```

Comment:
typo: ... but not all credential types (PasswordCredential and FederatedCredential) defined by the ...